Integrations/integration-opswat-metadefender.yml

```diff
@@ -39,42 +39,82 @@ configuration:
   defaultvalue: "66"
   type: 0
   required: true
+- display: It is cloud
+  name: cloud
+  defaultvalue: "false"
+  type: 8
+  required: false
 script:
   script: |
     var sendRequest = function(url,param, fileId) {
         var requestUrl = params.url + url + param;
```

Comment:
Please handle the case that the URL given in the parameters is followed by `/`

Reply:
Added